roles/openshift_logging/vars/main.yaml

```diff
@@ -0,0 +1,40 @@
+tr_or_ohlip: "{{ openshift_hosted_logging_openshift_logging_image_prefix or target_registry or none }}"
```

Comment:
I think this is a merge issue.  I dont think any vars/main.yaml should be available... or a majority of them should be removed.  I will check them all